backend/api/simulation.py

```
@@ -863,12 +851,29 @@ async def progress_to_next_scene(
+            ai_responses=scene_progress.ai_responses,
+            goal_achievement_score=scene_progress.goal_achievement_score,
+            interaction_quality=scene_progress.interaction_quality,
+            scene_feedback=scene_progress.scene_feedback,
+            started_at=scene_progress.started_at,
+            completed_at=scene_progress.completed_at,
             success=True,
             next_scene=next_scene_data,
             simulation_complete=False
         )
     else:
+        print(f"[DEBUG] /progress: No next_scene found, simulation_complete=True")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)